src/test/java/ladder/model/LineTest.java

```diff
@@ -0,0 +1,56 @@
+package ladder.model;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static ladder.model.LadderPath.*;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class LineTest {
+    @Test
+    @DisplayName("연속된 경로가 있다면 가로줄 생성에서 예외가 발생한다.")
```

Comment:
```suggestion
    @DisplayName("STAY 가 아닌 경로가 연속된다면 예외가 발생한다.")
```
그냥 제안인데 이 이름 어때요? DisplayName 안에는 도메인 관련 명칭 (STAY, LEFT, RIGHT, Ladder 등) 을 사용하지 않으려고 하시는 것 같은데 그 이유가 궁금해요!

Reply:
`/docs/README.md` 에 기능 요구사항을 작성할 때, 요구사항 명세서는 개발자 뿐만 아니라 모든 사람이 쉽게 이해할 수 있어야 한다고 생각해서 최대한 개발 관련 용어나 도메인 명칭을 사용하지 않으려고 했었는데, TDD 중 각 기능들을 하나씩 구현하다 보니 `@DisplayName` 안의 설명 또한 그렇게 따라갔던 것 같아요.

테스트코드도 '코드' 라는 말을 생각해 봤을 때 어노테이션 안에서는 오히려 도메인 관련 명칭을 직접 명시해 주는 게 좀 더 명확한 의미 전달이 가능하겠네요! 수정해 보겠습니다🙂